workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/DotUtil.java

```diff
@@ -122,9 +126,9 @@ public static StringBuffer getDot(Collection<OozieNode> oozieNodes) {
         return dot;
     }
 
-    public static StringBuffer getDashboardDot(Collection<OozieNode> oozieNodes, List<ProcessInfo> processInfoList) throws NullPointerException {
+    public static StringBuilder getDashboardDot(Collection<OozieNode> oozieNodes, List<ProcessInfo> processInfoList) throws NullPointerException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "getDashboardDot" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "oozieNodes". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)